### ISSUE TITLE
L'utilisateur peut consulter les photos d'un arrêt de bus.

### ISSUE THREAD

**Issue by maxbodin** [maxbodin/bus-stop-explorer#4]

**Comment by Bastien2203** [comment]
C'est les photos de tout les utilisateurs ?

**Comment by maxbodin** [comment]
Je propose d'afficher toutes les images de l'application par défaut, en ayant par exemple à droite une mosaïque d'images et à gauche la carte (en mobile : la carte en full et la mosaïque en window slider). Par contre quand on sélectionne un arrêt ou une ligne, on affiche uniquement les photos liées.

**Comment by Bastien2203** [comment]
Ok mais ma question c'était plutôt : toutes les photos sont publique quand on les post ?

**Comment by maxbodin** [comment]
Oui, à moins que tu ne vois un intérêt à ce que ce ne soit pas le cas ?

**Comment by the-me-0** [comment]
ça me paraît normal que les photos soient toutes publiques
